packages/grid/_modules_/grid/models/gridClasses.ts

```diff
@@ -0,0 +1,18 @@
+export interface GridClasses {
+  /**
+   * Styles applied to the root element.
+   */
+  root?: string;
+  /**
+   * Styles applied to the colCell element.
+   */
+  colCell?: string;
```

Comment:
Shouldn't it be `columnHeader` like the title of this PR? We were moving away from abbreviations in #1383.

Reply:
I can change the class name itself to match it. Either way works. If we are moving away from abbreviations then the path is clear.

Reply:
Ok, I've renamed the classes from `colCell` to 'columnHeader' for consistency and removing the use of abbreviation. This is now a breaking change PR, I'll list what those changes are in the PR description.